<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MilestoneConfig</class>
 <widget class="IConfigItem" name="MilestoneConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Milestone</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_Ip">
     <property name="text">
      <string>Ip address:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_Ip"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_User">
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_User"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_Password">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_Password"/>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>85</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IConfigItem</class>
   <extends>QWidget</extends>
   <header>IConfigItem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
